src/forms/FormAutocomplete.jsx

```diff
@@ -21,6 +21,7 @@ export function FormAutocomplete({
   filter,
   disabled: _disabled,
   afterChange,
+  allowCustomInput,
```

Comment:
@vbarcellos it will be better to rename this prop to `allowUnlistedValue`

Reply:
Changed